crates/cli/src/bin/wasm-bindgen.rs

```diff
@@ -83,7 +89,26 @@ struct Args {
                 If a bundler is used, it needs to be set up accordingly."
     )]
     split_linked_modules: bool,
-    input: PathBuf,
+    // The options below are deprecated. They're still parsed for backwards compatibility,
+    // but we don't want to show them in `--help` to avoid distracting users.
+    #[arg(long, hide = true)]
+    typescript: bool,
```

Comment:
Lets mark it as deprecated as well and leave a changelog entry for the corresponding CLI option.